src/facade/cmd_arg_parser.h

```diff
@@ -93,6 +88,10 @@ struct CmdArgParser {
     }
   }
 
+  // check next value ignore case and consume it
```

Comment:
1. Where would this be useful? Do we have many fixed arguments?
2. Let's call it `ExpectTag` or somehow to justify it's void type

Reply:
For DFLYCLUSTER GETSLOTINFO SLOTS ... we expect only SLOTS keyword